src/fedora_easyfix/templates/mainPage.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- For bootstrap-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```

Comment:
Please stick to using Bootstrap components as much as possible.

Reply:
With this being tagged onto the CDN, do you want bootstrap to be installed as a dependency in the repo? I'm not sure how you mean that here.